frontend/hub/namespaces/hooks/useHubNamespaceActions.tsx

```diff
@@ -44,6 +56,22 @@ export function useHubNamespaceActions(options?: {
         onClick: (namespace) =>
           pageNavigate(HubRoute.MyImports, { query: { namespace: namespace.name } }),
       },
+      {
+        type: PageActionType.Button,
+        selection: PageActionSelection.Single,
+        variant: ButtonVariant.primary,
+        icon: KeyIcon,
+        label: t(`Sign all collections`),
+        onClick: (namespace) =>
+          signCollection({
+            onComplete: options.onHubNamespacesSignAllCollections,
+            namespace: namespace,
+            signing_service: signing_service ?? '',
+          }),
+        isDisabled: () => (canSignAllCollections() ? '' : t('Select a repository filter')),
+        isHidden: () => !options.isDetailsPageAction,
```

Comment:
```suggestion
        isHidden: () => !options.isDetailsPageAction || !can_create_signatures,
```

Signing is still configurable, so we need these hidden when signing is off.

Reply:
However, I have it as: isHidden: () => !(options.isDetailsPageAction && canSignAllCollections()), which is: 

const canSignAllCollections = () => can_create_signatures && !can_upload_signatures;

Is this correct?

Reply:
yup, either `!options.isDetailsPageAction || !canSignAllCollections()` or `!(options.isDetailsPageAction && canSignAllCollections())` both make sense :+1: